### ISSUE TITLE
Hello, will you upload the training code later?

### ISSUE THREAD

**Issue by wzj-888** [google-research/omniglue#13]

**Comment by arjunkarpur** [comment]
Hi, we don't have any immediate plans to release training code. I would recommend you check out the [cvg/glue-factory](https://github.com/cvg/glue-factory) repo. I believe it would be straightforward to update the SuperGlue model/training code to re-implement OmniGlue.

We could eventually open-source the model definition, if there's interest.

**Comment by liniez** [comment]
> Hi, we don't have any immediate plans to release training code. I would recommend you check out the [cvg/glue-factory](https://github.com/cvg/glue-factory) repo. I believe it would be straightforward to update the SuperGlue model/training code to re-implement OmniGlue.
> 
> We could eventually open-source the model definition, if there's interest.

Hello,

Thank you for your outstanding work on OmniGlue! I am currently working on re-implementing the model and have a few questions. Could you please provide some details about the model definition? For example, are the keypoint locations normalized in the same way as in SuperGlue and LightGlue? What type of position encodings are used? Could you provide details about the MLP layers?

Thanks!

**Comment by wzj-888** [comment]
> Hi, we don't have any immediate plans to release training code. I would recommend you check out the [cvg/glue-factory](https://github.com/cvg/glue-factory) repo. I believe it would be straightforward to update the SuperGlue model/training code to re-implement OmniGlue.
> 
> We could eventually open-source the model definition, if there's interest.
Thank you, but I only see lightglue in glue-factory and I can't find superglue, and superglue has not released the training code, only the inference code. I am now deploying the glue-factory code and I have not succeeded. I don't know which model to choose as the baseline. I am a little confused. Do you have any suggestions? Thank you

**Comment by Sundragon1993** [comment]
@arjunkarpur Thank you for your great contribution to the community, We would be very happy if you could share the training code and suggestions on fine-tuning Omniglue to adapt with a specific domain.
